services/server/src/routes/availabilitySlot.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from "express";
+import availabilitySlot from "@/handlers/availabilitySlot";
+
+const router = Router();
+
+router.get("/", availabilitySlot.find);
+router.post("/set", availabilitySlot.set);
```

Comment:
[Optional] We can use `/` instead of `/set` because other routes follow the same pattern (constancy)

```suggestion
router.post("/", availabilitySlot.set);
```